R/copy.R

```diff
@@ -9,9 +9,10 @@
 file.move <- function(from, to, overwrite = FALSE) {
   stopifnot(file.exists(from))
   res <- suppressWarnings(file.rename(from = from, to = to))
-  if (.isFALSE(res)) {
+
+  if (!isTRUE(all(res))) {
     res2 <- file.copy(from = from, to = to, overwrite = overwrite)
-    if (isTRUE(res2)) {
+    if (isTRUE(all(res))) {
```

Comment:
Needs to be `res2` not `res`

Reply:
thius is what happens when you copy paste! sorry!